### PR TITLE
Adding the ability to hide login button from visitors pages 

### PR DESCRIPTION
**Pull request by desertblade**
This resolves issue #846. Decided to add it to the appearance page, mimicking the Github button functionality.

### CODE REVIEW

File: application/controllers/Options.php [hunk 1]
```diff
@@ -117,7 +117,7 @@ function appearance_save() {
 			if($public_maps_update_status == TRUE) {
 				$this->session->set_flashdata('success', __("Options saved"));
 			}
-
+
```
Comment (phl0) [inline application/controllers/Options.php:120]: Please remove this stray whitespace

File: application/views/options/appearance.php [hunk 2]
```diff
@@ -103,6 +103,15 @@
                             <small id="publicGithubButtonHelp" class="form-text text-muted"><?= __("This enables the button to Wavelog's Github page in the public view"); ?></small>
                         </div>
 
+                        <div class="mb-3">
+                            <label for="publicLoginButton"><?= __("Public Login Button"); ?></label>
+                            <select class="form-select" id="publicLoginButton" name="publicLoginButton" aria-describedby="publicLoginButtonHelp" required>
+                                <option value='true' <?php if($this->optionslib->get_option('public_login_button') == "true") { echo "selected=\"selected\""; } ?>><?= __("Enabled"); ?></option>
+                                <option value='false' <?php if($this->optionslib->get_option('public_login_button') == "false") { echo "selected=\"selected\""; } ?>><?= __("Disabled"); ?></option>
+                            </select>
+                            <small id="publicLoginButtonHelp" class="form-text text-muted"><?= __("This enables the button to login to Wavelog in the public/vistors view"); ?></small>
```
Comment (phl0) [inline application/views/options/appearance.php:112]: Please correct typo here :) (->vistors)

**Review by phl0 — CHANGES_REQUESTED** [review]
Found two small things to change before we can accept this PR. 

But please also change it to be created against dev branch instead of master. Master branch is only used for stable code to release. All development happens in the dev branch.

**Comment by desertblade** [comment]
Should be good to go. Its what I get by rushing.

**Review by phl0 — APPROVED** [review]
Fine now. So merge time 😇

**Pull request merged**